docs/Calamari/Node Operators/get_calamari.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 2
+---
+
+# Get Calamari
+
+## Choose your installation variant
+
+### Option 1: Build From Source
+
+Use this option if you are comfortable working with Cargo and Rust source code and have time or a fast multi-core processor for compiling.
+
+The following instructions are for latest Ubuntu distribution.
+
+```bash
+# install build deps
+sudo apt-get update
+sudo apt-get -y install build-essential clang curl git libssl-dev make pkg-config
+
+# install rust toolchain
+curl -s https://sh.rustup.rs -sSf | sh -s -- -y
```

Comment:
What I've generally seen in documentation is linking to the install page of the tool, in case it changes/to let people verify the commands. We do have a bit more setup for Rust than is perhaps usual, because of nightly and wasm, but I think we should probably link/explain the steps more.

Reply:
I guess we can use this one ---> https://substrate.dev/docs/en/knowledgebase/getting-started/